coreos-base/coreos-init/coreos-init-9999.ebuild

```diff
@@ -3,14 +3,14 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="4"
-CROS_WORKON_PROJECT="coreos/init"
+CROS_WORKON_PROJECT="sdemos/init"
 CROS_WORKON_LOCALNAME="init"
 CROS_WORKON_REPO="git://github.com"
 
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="c48b1fdd52fa78c7e67db33861dabaa93f3e73e1"
+	CROS_WORKON_COMMIT="ed478168f0133cc140baa08d4d77e4333e8991d7"
```

Comment:
This would need to bump the stable ebuild version.  (And obviously reference the merge commit in the real `init` repo when ready.)